cluster-autoscaler/cloudprovider/aws/aws_wrapper.go

```diff
@@ -780,3 +784,17 @@ func buildLaunchTemplateFromSpec(ltSpec *autoscaling.LaunchTemplateSpecification
 		version: version,
 	}
 }
+
+func taintEksTranslator(t *eks.Taint) (apiv1.TaintEffect, error) {
+	// Translation between AWS EKS and Kubernetes taints
```

Comment:
Can you add a link to the AWS docs here?

Reply:
Here https://docs.aws.amazon.com/eks/latest/APIReference/API_Taint.html, @wcarlsen

Reply:
Very good idea. Thanks